src/syscheckd/src/db/tests/db/ComponentTest/registryInterface/registryTest.cpp

```diff
@@ -0,0 +1,153 @@
+/*
+ * Wazuh Syscheck
+ * Copyright (C) 2015-2021, Wazuh Inc.
+ * December 31, 2021.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation.
+ */
+
+#include "registryTest.h"
+#include "db.h"
+#include "db.hpp"
+
+
+const auto insertRegistryKeyStatement1 = R"({
+        "checksum":"a2fbef8f81af27155dcee5e3927ff6243593b91a", "gid":0, "group_name":"root", "arch":1,
+        "last_event":1596489275, "mode":0, "mtime":1578075431, "path":"HKEY_LOCAL_MACHINE\\SOFTWARE\\regtest1", "perm":"-rw-rw-r--",
+        "scanned":1, "uid":0, "user_name":"fakeUser"
+    }
+)"_json;
+
+const auto insertRegistryValueStatement1 = R"({
+        "name":"testRegistry1", "scanned":1, "last_event":1596489275, "checksum":"a2fbef8f81af27155dcee5e3927ff6243593b91a",
+        "mode":0, "size":4925, "type":0, "hash_md5":"4b531524aa13c8a54614100b570b3dc7",
+        "hash_sha1":"7902feb66d0bcbe4eb88e1bfacf28befc38bd58b",
+        "hash_sha256":"e403b83dd73a41b286f8db2ee36d6b0ea6e80b49f02c476e0a20b4181a3a062a",
+        "arch":0, "path":"HKEY_LOCAL_MACHINE\\SOFTWARE\\regtest1"
+    }
+)"_json;
+
+const auto insertRegistryKeyStatement2 = R"({
+        "checksum":"a2fbef8f81af27155dcee5e3927ff6243593b91a", "gid":0, "group_name":"root", "arch":1,
+        "last_event":1596489275, "mode":0, "mtime":1578075431, "path":"HKEY_LOCAL_MACHINE\\SOFTWARE\\regtest2", "perm":"-rw-rw-r--",
+        "scanned":1, "uid":0, "user_name":"fakeUser"
+    }
+)"_json;
+
+const auto insertRegistryValueStatement2 = R"({
+        "name":"testRegistry2", "scanned":1, "last_event":1596489275, "checksum":"a2fbef8f81af27155dcee5e3927ff6243593b91a",
+        "mode":0, "size":4925, "type":0, "hash_md5":"4b531524aa13c8a54614100b570b3dc7",
+        "hash_sha1":"7902feb66d0bcbe4eb88e1bfacf28befc38bd58b",
+        "hash_sha256":"e403b83dd73a41b286f8db2ee36d6b0ea6e80b49f02c476e0a20b4181a3a062a",
+        "arch":0, "path":"HKEY_LOCAL_MACHINE\\SOFTWARE\\regtest2"
+    }
+)"_json;
+
+void transaction_callback(ReturnTypeCallback resultType, const cJSON* result_json, void* user_data){}
+
+TEST_F(RegistryTestWinFixture, TestFimDBGetCountRegistryEntry)
```

Comment:
You could use DBTestWinFixture inside dbTest.h file to avoid duplicate code

Reply:
done